.github/workflows/check.yml

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Run Checks
```

Comment:
Run checks is too generic - Maybe 'Run Formatting and Header Checks' ?

Reply:
This was a deliberate choice to make it look nice in the check ui by avoding duplication: 
![grafik](https://github.com/facebookincubator/velox/assets/16141871/01080789-e6a3-4b11-9a40-1e7530585f8d)

+ we likely will add other checks that would require an even longer title